include/igl/fit_cubic_bezier.cpp

```diff
@@ -22,9 +22,12 @@ IGL_INLINE void igl::fit_cubic_bezier(
   // Don't attempt to fit curve to single point
   if(nPts==1) { return; }
   // Avoid using zero tangent
-  const static auto tangent = [&](const int i,const int dir)->Eigen::RowVectorXd
+  const static auto tangent = [](
```

Comment:
Why do you even need the lambda to be static? It wouldn't make a difference to the compiler.